db/migrate/20200211225056_modify_offers_table.rb

```diff
@@ -0,0 +1,87 @@
+class ModifyOffersTable < ActiveRecord::Migration[5.2]
+  def up
+    change_table :offers, bulk: true do |t|
+      t.rename(:photography_cents, :photography_info)
+      t.rename(:shipping_cents, :shipping_info)
+      t.rename(:insurance_cents, :insurance_info)
+      t.rename(:other_fees_cents, :other_fees_info)
+
+      t.change(
+        :photography_info,
+        :string,
+        using:
+          'CASE WHEN (photography_info IS NOT NULL) ' \
+            "THEN format('%s %s', TRIM(to_char(photography_info / 100, '99999')), currency) END"
+      )
+      t.change(
+        :shipping_info,
+        :string,
+        using:
+          'CASE WHEN (shipping_info IS NOT NULL) ' \
+            "THEN format('%s %s', TRIM(to_char(shipping_info / 100, '99999')), currency) END"
+      )
+      t.change(
+        :insurance_info,
+        :string,
+        using:
+          'CASE WHEN (insurance_info IS NOT NULL) ' \
+            "THEN format('%s %s', TRIM(to_char(insurance_info / 100, '99999')), currency) " \
+            'WHEN (insurance_percent IS NOT NULL) ' \
+            "THEN TRIM(TO_CHAR(insurance_percent * 100, '99D00%')) END"
+      )
+      t.change(
+        :other_fees_info,
+        :string,
+        using:
+          'CASE WHEN (other_fees_info IS NOT NULL) ' \
+            "THEN format('%s %s', TRIM(to_char(other_fees_info / 100, '99999')), currency) " \
+            'WHEN (other_fees_percent IS NOT NULL) ' \
+            "THEN TRIM(TO_CHAR(other_fees_percent * 100, '99D00%')) END"
```

Comment:
👍 Curious if doing this in SQL is indeed the ideal way? Or if a view helper would be a better place? I guess in this case it does make all the data consistent, but also a little less flexible. (That said, I know very little SQL.)